common/build/build-common/tsdoc-base.json

```diff
@@ -0,0 +1,28 @@
+{
+	"$schema": "https://developer.microsoft.com/json-schemas/tsdoc/v0/tsdoc.schema.json",
+
+	// Include the definitions that are required for API Extractor
+	"extends": ["@microsoft/api-extractor/extends/tsdoc-base.json"],
+
+	"tagDefinitions": [
+		{
+			// This tag indicates API is part of a legacy API set.
+			"tagName": "@legacy",
+			"syntaxKind": "modifier",
+			"allowMultiple": true
```

Comment:
I think this can be false? There shouldn't be a use for specifying this multiple times. Same with `@system`.